easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -246,7 +303,7 @@ def configure_step(self):
 
         # move non-espresso directories to where they're expected and create symlinks
         try:
-            dirnames = [d for d in os.listdir(self.builddir) if not d == self.install_subdir]
+            dirnames = [d for d in os.listdir(self.builddir) if d != self.install_subdir and d != 'd3q-latest']
```

Comment:
@akesandgren Style nitpicking: this is slightly cleaner (I overlooked that earlier):

```python
dirnames = [d for d in os.listdir(self.builddir) if d not in [self.install_subdir, 'd3q-latest']]